mkosi

```diff
@@ -690,6 +690,16 @@ def mkfs_btrfs(label: str, dev: str) -> None:
     run(["mkfs.btrfs", "-L", label, "-d", "single", "-m", "single", dev], check=True)
 
 
+def mkfs_generic(args: CommandLineArguments, label: str, mount: str, dev: str) -> None:
+
```

Comment:
annoying extra blank line that just got removed in a recent commit from other functions. So we may want to avoid adding them on new commits.